server/routes/student.ts

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { putStudentHealth } from '../controllers';
+
+const studentRouter = Router();
+
+studentRouter.put('/:studentId/health', putStudentHealth);
```

Comment:
The parent only can update the health, so pass the parent middleware and check if this student is his child or not.